integrations/whatsapp/src/index.ts

```diff
@@ -187,7 +192,7 @@ const integration = new bp.Integration({
 
     const secret = getSecret(ctx)
     // For testing purposes, if you send the secret in the header it's possible to disable signature check
-    if (req.headers['x-secret'] !== secret) {
+    if (secret?.length && req.headers['x-secret'] !== secret) {
```

Comment:
nit: `if (secret?.length` can be expressed as simply `if (secret` (for readability) as `secret` is a string?

Reply:
In theory, they could have a clientSecret set as an empty string, but I modified the function to fit this implementation